### PR TITLE
Add Mac compatibility to boulder-start

### PR DESCRIPTION
**Pull request by BKreisel**
The version of sort that ships with OS X does not support the -V
version flag. Emulate that functionality with some sed-fu

### CODE REVIEW

**Comment by BKreisel** [comment]
Initial steps in attempt to get integration tests up and running as requested by @jmhodges.

I'm also considering writing a script to install integration dependencies such as libtool-ltdl, mardiadb, etc... on Mac. Any interest in this?

File: tests/boulder-start.sh [hunk 1]
```diff
@@ -4,11 +4,16 @@
...
 # ugh, go version output is like:
 # go version go1.4.2 linux/amd64
-GOVER=`go version | cut -d" " -f3 | cut -do -f2`  
+GOVER=`go version | cut -d" " -f3 | cut -do -f2`
 
 # version comparison
 function verlte {
+  if [ `uname` == 'Darwin' ]; then
+    [ "$1" = "`echo -e \"$1\n$2\" |  sed 's/\b\([0-9]\)\b/0\1/g' \
+      | sort | sed 's/\b0\([0-9]\)/\1/g' | head -n1`" ]
```
Comment (kuba) [inline tests/boulder-start.sh]: this is so complicated that it's worth to put any inline comment, at least "`sort -V` doesn't work on Mac"

**Comment by kuba** [comment]
I think this is generally OK to merge. Thanks!

**Comment by bmw** [comment]
lgtm

Thanks again for the help @BKreisel!

**Pull request merged**